ccd-definition/CaseEventToFields/CareSupervision/uploadDocuments.json

```diff
@@ -137,11 +137,10 @@
     "CaseEventID": "uploadDocuments",
     "CaseFieldID": "documents_socialWorkOther",
     "PageFieldDisplayOrder": 12,
-    "DisplayContext": "OPTIONAL",
+    "DisplayContext": "COMPLEX",
     "PageID": 1,
     "PageDisplayOrder": 1,
-    "PageColumnNumber": 1,
-    "ShowSummaryChangeOption": "Y"
```

Comment:
Accidental removal of the ShowSummaryChangeOption?

Reply:
accidental. I will add that back.